docs/README.rst

```diff
@@ -88,33 +88,33 @@ configures the development environment.
 This state will install Eclipse from archive only.
 
 ``eclipse.macapp``
-^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^
 
 This state will install Eclipse from source only.
 
 ``eclipse.config``
-^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^
 
 This state will configure npmrc and/or environment and has a dependency on ``eclipse.install``
 via include list.
 
 ``eclipse.plugins``
-^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^
 
 This state will install Eclipse plugins.
 
 ``eclipse.developer``
-^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^
```

Comment:
For the heading symbols, they should extend the whole length of the heading.  So in this particular case, the original value was already correct.  As for the rest, they should be adjusted in the right way.  I'll see if I can fix that as well.